README.md

```diff
@@ -2,4 +2,107 @@
 
 [![License badge](https://img.shields.io/badge/license-GPL-blue.svg)](https://opensource.org/licenses/GPL-3.0)
 [![Docker badge](https://img.shields.io/docker/pulls/dojot/iotagent-json.svg)](https://hub.docker.com/r/dojot/iotagent-json/)
-[![Support badge]( https://img.shields.io/badge/support-sof-yellowgreen.svg)](http://stackoverflow.com/questions/tagged/dojot)
+[![Travis badge](https://travis-ci.org/dojot/iotagent-json.svg?branch=cpqd_master)](https://travis-ci.org/dojot/iotagent-json#)
+
+IoT agents are responsible for receiving messages from physical devices (directly or through a gateway) and sending them commands in order to configure them. This iotagent-json, in particular, receives messages via MQTT with JSON payloads.
+
+## How does it work
+The iotagent-json depends on two things: a Kafka broker, so that it can receive messages informing it about new devices (and, in extension, about their updates and removals), and, obviously, a MQTT broker, so that it can receive messages from the devices. It waits for messages sent through these two elements: from device manager with a management operation on a device and from the MQTT broker with a message sent by a device.
```

Comment:
>, and, obviously, a MQTT broker,

'obviously' sounds very presumptuous, maybe something like 'as expected' would sound better

> from device manager

from 'the' device-manager